dev-tools/ecs-migration.yml

```diff
@@ -29,7 +29,10 @@
   alias: true
 
 - from: source
-  to: ["log.file.path", "log.source.ip"]
+  to:
+  - log.file.path
+  - log.source.ip
```

Comment:
I assume we can remove  `log.source.ip` from this list?

Reply:
Not sure, I kept it in case it was being used, or was going to be used in other places, but I have no problem with removing it.

Reply:
It "should" not be used anywhere else. Can you also do a quick search to make sure this is the case and remove it?

Reply:
Sure :+1:

Reply:
Should I also remove it from `fields.common.yml`?